xml/admin_ceph_upgrade.xml

```diff
@@ -263,8 +263,34 @@
 &prompt.smaster;salt '*' saltutil.sync_all
 &prompt.cephuser;ceph -s
 </screen>
+   <sect3 xml:id="upgrade-disable-insecure">
+    <title>Disable insecure clients</title>
+    <para>
+     Since &prevcephname; v4.2.20, a new health warning was introduced that
```

Comment:
The version number should have been `v14.2.20` here?

Reply:
The question is: do we want to consolidate and disable secure clients still on SES6 before upgrading to SES7, or after the upgrade? i vote for the former...

Reply:
@tbazant In the commit I see `v4.2.20` but that looks like a typo - seems like a `1` is missing between `v` and `4`?

Reply:
yes, correct, going to fix now......done